### ISSUE TITLE
Further Caveats

### ISSUE THREAD

**Issue by josephgupkim** [coffeeNcake/bwa-u4-feature-request#3]
The Caveats section was a bit lacking for me as I'm sure you could of added more?

"To avoid any confusion, the enter key only triggers the search when the user is in the search input field."

How would you check if the user is currently in the search input field? Would you still allow a search to be triggered if the input value was an empty string? If so, would it return no results or all results?

**Comment by coffeeNcake** [comment]
Yes, I agree there could have been more caveats.  The onChange event handler would only trigger on the field that the user is on and no, I wouldn't allow a search to be triggered if the input value was empty.
